handlers/main.yml

```diff
@@ -1,11 +1,11 @@
 ---
 
 - name: mongodb reload
-  service: name={{ mongodb_daemon_name }} state=reloaded
+  service: name={{ mongodb_service }} state=reloaded
```

Comment:
If you want to use mongodb_service instead of daemon name you should update `mongodb_service` variable.
`"{{ 'mongod' if ('mongodb-org' in mongodb_package) else 'mongodb' }}"`